concourse/pipelines/deploy.yml

```diff
@@ -16,6 +16,14 @@ resource_types:
       repository: governmentpaas/s3-resource
       tag: latest # TODO - don't use latest (once we've worked out a policy for third party images)
 
+  - name: registry-image-v2
```

Comment:
Why `registry-image-v2`? It appears to be v1 (1.2.1). If this is to distinguish it from the anchor called `registry-image` below then it'd be much better to rename the anchor (say to `image`) and then use the expected name for this resource so that it's consistent with the others (i.e. named after its repo, minus the `-resource` part).

If you wanna do that in a follow up then that's fine.

Reply:
I have overridden the default resource type now as discussed in slack. Thanks